backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/work/builder/factory/impl/Elasticsearch6WorkBuilderFactory.java

```diff
@@ -63,14 +63,11 @@
 
 import com.google.gson.JsonObject;
 
-/**
- * @author Yoann Rodiere
- */
-public class ElasticsearchWorkBuilderFactoryImpl implements ElasticsearchWorkBuilderFactory {
+public class Elasticsearch6WorkBuilderFactory implements ElasticsearchWorkBuilderFactory {
```

Comment:
So what if you use the same for 7 too? It will be named 6 but will also be used by 5 and 7?

Reply:
OK, makes sense.